splunk_connect_for_snmp_poller/manager/poller.py

```diff
@@ -55,7 +56,8 @@ def run(self):
         counter = 0
         while True:
             if counter == 0:
-                self.check_inventory()
+                # self.check_inventory()
```

Comment:
instead of a comment, please just remove the code

Reply:
removed